web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx

```diff
@@ -0,0 +1,138 @@
+
+:::info
+This tutorial is aimed at beginners that already know how to publish events,
+if you're not sure if that is you, take a look at the [previous tutorial](./first-event).
+:::
+
+Before we start, make sure Actyx is running! If you're unsure how to do so,
+you can review how to start Actyx in our [how-to guide](/docs/how-to/local-development/install-actyx#starting-and-stopping-actyx).
+
+To query Actyx you can either use the CLI or the Node Manager —
+don't worry, we will cover both in this tutorial.
+
+## Querying Actyx using the CLI
+
+If you're familiar with CLI applications, it won't be a surprise that the interface to
+query events, is similar to the interface to publish them.
+Let's take a look into the help text!
+
+```
+USAGE:
+    ax events query [FLAGS] [OPTIONS] <NODE> <query>
+
+FLAGS:
+    -h, --help       Prints help information
+    -j, --json       Format output as JSON
+    -V, --version    Prints version information
+    -v
+
+OPTIONS:
+    -i, --identity <FILE>
+            File from which the identity (private key) for
+            authentication is read
+
+ARGS:
+    <NODE>     the IP address or <host>:<admin port> of the node
+               to perform the operation on
+    <query>    event API query (read from file if the argument
+               starts with @)
+```
+
+As you can see, we still need the node, but instead of a `payload`, we're sending a `query` now.
+
+The query is required to be valid AQL code, which we will cover in depth in a later tutorial.
+For this case, we will provide the required AQL query and [explain what it does](#what-does-the-query-do)!
+
+### Running the query
+
+Now that you're up to speed, the command we will be running is the following:
+
+<Tabs>
+<Windows>
+
+```
+.\ax events query localhost "FROM allEvents"
+```
+
+</Windows>
+
+<Linux>
+
+```
+./ax events query localhost "FROM allEvents"
+```
+
+</Linux>
+
+<Mac>
+
+```
+./ax events query localhost "FROM allEvents"
+```
+
+</Mac>
+</Tabs>
+
+As a result, your output should be similar to:
+
+```
+0/z4tg-0@2023-08-08T15:57:34.760969+01:00: {"streamName": "default", "streamNr": 0}
+1/z4tg-0@2023-08-08T15:57:34.763995+01:00: {"streamName": "discovery", "streamNr": 1}
+2/z4tg-0@2023-08-08T15:57:34.766578+01:00: {"streamName": "metrics", "streamNr": 2}
+3/z4tg-0@2023-08-08T15:57:34.769054+01:00: {"streamName": "files", "streamNr": 3}
+4/z4tg-0@2023-08-08T15:57:34.772122+01:00: {"NewListenAddr": ["12D3KooWSokZosGoYhKa5GxVjuDFoY2fPh7SLF6TcTS3bwGZhRzs", "/ip4/192.168.0.104/tcp/4001"]}
+5/z4tg-0@2023-08-08T15:57:34.774454+01:00: {"PinAdded": {"app_id": "com.actyx", "query": "FEATURES(zøg aggregate timeRange) FROM (((isLocal & appId(com.actyx)) & \'files:created\') & TIME >= 2023-08-08T02:57:34Z) SELECT _.cid END", "duration": {"secs": 604800, "nanos": 0}}}
+6/z4tg-0@2023-08-08T15:57:43.737290+01:00: "hello"
+```
+
+:::info
+You may have more events in your output, don't worry, that is completely normal!
```

Comment:
When switching to tag `'a'`, this note should only say that with `FROM allEvents` you can also see all the Actyx internal events if you want to.